package.lisp

```diff
@@ -19,9 +19,20 @@
    #:*parse-json-booleans-as-symbols*
    #:*parse-json-null-as-keyword*
 
+   
+   #:*allow-nan*     
+   #:*yason-float-parser* 
+   #:*yason-float-type*   
+   #:*allow-loose-floats*
+
+
+
    #:true
    #:false
    #:null
+   #:nan
```

Comment:
About `:NAN` and ±∞ I'd like to see what CSR is doing about them - we'll want to be compatible. (See http://www.sbcl.org/sbcl20/ for the one-line note.)